server/routes/search.ts

```diff
@@ -24,12 +24,19 @@ export default function routes(
     )
 
   get(['/prisoner', '/prisoner-visit'], (req, res) => {
+    const isVisit = req.originalUrl.includes('-visit')
     const search = req?.body?.search
-
+    let establishmentHref = ''
```

Comment:
Better to default to `'/search/prisoner/'` and only change on the `if` condition